internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -1806,8 +1886,9 @@ func NewStorageContainer(cr *model.CryostatInstance, imageTag string, tls *TLSCo
 	var containerSc *corev1.SecurityContext
 	envs := []corev1.EnvVar{
 		{
-			Name:  "CRYOSTAT_BUCKETS",
-			Value: "archivedrecordings,archivedreports,eventtemplates,probes",
+			Name: "CRYOSTAT_BUCKETS",
+			// TODO use cr.Spec.ObjectStorageOptions.StorageBucketNameOptions if specified, with the list below as default values
+			Value: "archivedrecordings,archivedreports,eventtemplates,probes,heapdumps,threaddumps",
```

Comment:
Should this be implemented before merging?

Reply:
Would it make sense to update the CRD description to mention that the StorageBucketNameOptions only apply to external providers, and remove this TODO?